test_conformance/gl/common.h

```diff
@@ -32,7 +32,8 @@ struct format {
 };
 
 // These are the typically tested formats.
-
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-variable"
```

Comment:
I'm not familiar with the gl tests, so it's not clear to me why we are ignoring `-Wunused-variable` here, as from a quick search `common_formats` &  `depth_formats` seem to be used in the tests. Could you stick the reasoning in a comment?

Reply:
I think the main issue is that these variables aren't `const` and not every .cpp file is using both variables, hence they trigger unused-variable warnings for some .cpp files.  Making these variables `const` requires propagating the constness to other places, which I'd rather leave outside of the scope of PR.  

I've clarified this in a comment.

Reply:
Followup adding `const` is here: https://github.com/KhronosGroup/OpenCL-CTS/pull/1493